app/views/hyku_addons/account_settings/settings/_render_select.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render 'hyku_addons/account_settings/settings/partial_for_select',
```

Comment:
Should this be `partial_for_select_fields`?  I don't see a `partial_for_select` anywhere.

Reply:
Well spotted. This file is actually not required at all and must have been left over from my first attempt and working out what was going on.